frontend/packages/console-app/src/components/volume-modes/volume-mode.tsx

```diff
@@ -20,14 +20,15 @@ export const VolumeModeSelector: React.FC<VolumeModeSelectorProps> = (props) =>
     storageClass,
     availableVolumeMode,
     loaded,
+    volMode,
```

Comment:
nitpick: Could we name this something similar to the above suggestions, like `defaultVolumeMode` or `initialVolumeMode`?

Reply:
+1 changed to `initialVolumeMode`